update-xars.sh

```diff
@@ -0,0 +1,49 @@
+VERSION=$1
+DIR=$2
+
+REPO=https://exist-db.org/exist/apps/public-repo/public
```

Comment:
This in turn should be made available as a build ARG in the Dockerfile, so we won't need to update this script when the URL changes and we have all parameters in one place.